src/uct/base/uct_iface.c

```diff
@@ -911,3 +921,19 @@ uct_base_ep_connect_to_ep(uct_ep_h tl_ep,
 
     return uct_ep_connect_to_ep_v2(tl_ep, device_addr, ep_addr, &param);
 }
+
+ucs_status_t
+uct_iface_is_reachable_v2_wrapper(const uct_iface_h tl_iface,
+                                  const uct_device_addr_t *dev_addr,
+                                  const uct_iface_addr_t *iface_addr,
+                                  uct_iface_is_reachable_v2_func_t iface_is_reachable_v2)
```

Comment:
we should have this in iface->ops, so just call 
```
uct_iface_is_reachable_v2(tl_iface, &params);
```
at line 938

Reply:
Great! Thanks, Evgeny.

Reply:
Looks it doesn't work. Some testcase will failed.
```
/scrap/azure/agent-06/AZP_WORKSPACE/2/s/contrib/../src/uct/ib/dc/dc_mlx5.c: [ uct_dc_mlx5_iface_is_reachable_v2() ]
      ...
      898     uct_dc_mlx5_iface_t UCS_V_UNUSED *iface;
      899 
      900     iface = ucs_derived_of(tl_iface, uct_dc_mlx5_iface_t);
==>   901     ucs_assert_always(iface_addr != NULL);

```

Looks use should call the same level v2 function rather than the upper layer's.